src/App.js

```diff
@@ -23,25 +24,39 @@ axios.defaults.baseURL = '/';
  *
  * @type {React.FunctionComponent}
  */
-export const App = () => (
-  <Router>
-    <ContextProvider>
-      <Navbar />
-      <div className='container'>
-        <Switch>
-          <Route exact path='/' component={HomePage} />
-          <Route path='/login' component={LoginPage} />
-          <Route path='/signup' component={RegisterPage} />
-          <Route path='/settings' component={SettingPage} />
-          <Route path='/notification' component={NotificationPage} />
-          <Route exact path='/u/:userHandle' component={UserPage} />
-          <Route
-            exact
-            path='/u/:userHandle/post/:postId'
-            component={UserPage}
-          />
-        </Switch>
-      </div>
-    </ContextProvider>
-  </Router>
-);
+export const App = () => {
+  const AppContent = () => (
+    <Router>
+      <ContextProvider>
+        <Navbar />
+        <div className='container'>
+          <Switch>
+            <Route exact path='/' component={HomePage} />
+            <Route path='/login' component={LoginPage} />
+            <Route path='/signup' component={RegisterPage} />
+            <Route path='/settings' component={SettingPage} />
+            <Route path='/notification' component={NotificationPage} />
+            <Route exact path='/u/:userHandle' component={UserPage} />
+            <Route
+              exact
+              path='/u/:userHandle/post/:postId'
+              component={UserPage}
+            />
+          </Switch>
+        </div>
+      </ContextProvider>
+    </Router>
+  );
+  return (
+    <div>
+      <Helmet>
+        <title>SocMon</title>
+        <meta
+          name='description'
+          content='SocMon is a place to meet and communicate with friends and family. Create posts, then comment and like on what others have posted.'
+        />
```

Comment:
You should put helmet in a separate component, and also, you can useLocation to get what page you're on and update the title to be SocMon - whatever page title

Reply:
I'll attempt to do that at a later date. At least at this time, that's a pretty low priority compared to the rest of the actions in the project board.

Reply:
fixed